src/lib/widget/_utilities/api.utilities.ts

```diff
@@ -205,6 +205,9 @@ export function widgetApiFactory(componentApi: ReturnType<typeof _componentApi>)
           ...startOptions?.forgerock,
           // Only include a `tree` property if the `journey` options prop is truthy
           ...(startOptions?.journey && { tree: startOptions?.journey }),
+          ...(!startOptions?.recaptchaAction
+            ? { recaptchaAction: startOptions?.journey }
+            : { recaptchaAction: startOptions.recaptchaAction }),
```

Comment:
Can we bring this logic down into the `journey.store`? I feel like you [already have it there](https://github.com/ForgeRock/forgerock-web-login-framework/pull/252/files#diff-e01603033c52bf6a043d215ff082252220802c8f537131e59b6fc3eb8de26cd3R402), so this feels a bit redundant. Also, the way you have it here means the dev can't utilize the other ways of passing in a journey or tree value. There's the configuration option and the journey initialization option. As it is, the dev cannot leverage the other options for declaring the tree/journey.